data-prepper-plugins/opensearch-api-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearchapi/OpenSearchAPISourceConfig.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.opensearchapi;
+
+import org.opensearch.dataprepper.http.BaseHttpServerConfig;
+
+public class OpenSearchAPISourceConfig extends BaseHttpServerConfig {
+
+    static final String DEFAULT_ENDPOINT_URI = "/";
+    static final int DEFAULT_PORT = 9202;
```

Comment:
Is there a particular reason to default to 9202 rather than 9200? I think in most cases, the community will run this on a different machine than OpenSearch. A default of 9200 would let it more easily stand in for OpenSearch.

Reply:
Thanks @dlvenable. I will resolve this.